### DISCUSSION TITLE
jk_b2a8s20p con mqtt . ble , reconet

### DISCUSSION THREAD

**Discussion by calasci** [syssi/esphome-jk-bms#494] [General]
I have the esp32 with bluetooth connected to the jk b2a8s20p with mqtt, but it continually disconnects, it freezes, thanks

**Comment by syssi** [comment]
Could you describe your setup (which ESP32 model / dev board do you use, what's the distance to your BMS and WiFi accesspoint) and provide some logs? If you attach a computer to the serial interface of the ESP32 you can see log messages even if the device isn't connected to the WiFi anymore or crashed.

**Reply by calasci** [reply]
esp32wroom , wifi 1 metro , vedo che non si collega al bluetoot del bms ,   grazie
